src/inventory/reports.py

```diff
@@ -73,4 +79,106 @@ def execute(self, report_file_name: str) -> str:
 
         _logger.info(f"completed file upload")
 
-        return f"https://{target_bucket}.s3.amazonaws.com/{report_s3_key}"
\ No newline at end of file
+        return f"https://{target_bucket}.s3.amazonaws.com/{report_s3_key}"
```

Comment:
I highly doubt that this URL will work for Govcloud. You'll have to actually get it from the S3 instead of generating yourself. Example of S3 object URL in govcloud - `https://${target_bucket}.s3-us-gov-west-1.amazonaws.com/${key}`